jobserver/middleware.py

```diff
@@ -64,3 +67,34 @@ def __call__(self, request):
         response.headers.setdefault("X-XSS-Protection", "1; mode=block")
 
         return response
+
+
+class ClientAddressIdentification:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        ip = request.META["REMOTE_ADDR"]
```

Comment:
Are you sure - REMOTE_ADDR is not a header, its part of the default WSGI environ?

Reply:
TIL!  It might be worth adding a comment about that, getting WSGI environ values is pretty uncommon IME so making it clear that's what you're doing rather than getting a header would be 👍